docs/source/markdown/podman-volume-create.1.md

```diff
@@ -46,13 +46,16 @@ The `type` option sets the type of the filesystem to be mounted, and is equivale
 The `device` option sets the device to be mounted, and is equivalent to the `device` argument to **mount(8)**.
 The `copy` option enables copying files from the container image path where the mount is created to the newly created volume on the first run.  `copy` is the default.
 
-The `o` option sets options for the mount, and is equivalent to the `-o` flag to **mount(8)** with these exceptions:
+The `o` option sets options for the mount, and is equivalent to the filesystem
+options (also `-o`) passed to **mount(8)** with the following exceptions:
 
   - The `o` option supports `uid` and `gid` options to set the UID and GID of the created volume that are not normally supported by **mount(8)**.
   - The `o` option supports the `size` option to set the maximum size of the created volume, the `inodes` option to set the maximum number of inodes for the volume and `noquota` to completely disable quota support even for tracking of disk usage. Currently these flags are only supported on "xfs" file system mounted with the `prjquota` flag described in the **xfs_quota(8)** man page.
   - The `o` option supports using volume options other than the UID/GID options with the **local** driver and requires root privileges.
   - The `o` options supports the `timeout` option which allows users to set a driver specific timeout in seconds before volume creation fails. For example, **--opts=o=timeout=10** sets a driver timeout of 10 seconds.
 
+***Warning*** Do not confuse the `--opts,-o` create option with the `-o` mount option.  For example, with `podman volume create`, use `-o=o=uid=1000` *not* `-o=uid=1000`.
```

Comment:
Maybe add an example instead of this warning?

Reply:
I think there is one already, but this is such an easy mistake for humans to make (since `-o` == `-o`) I thought a big-fat warning might be help avoid some headaches.  However, I'm bias in my opinion since this resulted in my hair-loss for a week or so.  Would it be better if  I toned it down to a `***Note***` instead of a warning?

Reply:
Sure

Reply:
Done and pushed.